libs/shared/test-utils/src/lib/editor-page-object.ts

```diff
@@ -46,8 +46,12 @@ export abstract class EditorPageObject<T> extends PageObject<T> {
     this.clickElement(this.queryPo.copyBtn);
   }
 
-  setSelectValueById(inputId: string, value: number): void {
-    this.setInputValueById(inputId, `${value}: ${value}`);
+  setSelectValueById(inputId: string, value: string | number): void {
+    if (typeof value === 'string') {
+      this.setInputValueById(inputId, `${value}`);
+    } else {
+      this.setInputValueById(inputId, `${value}: ${value}`);
+    }
```

Comment:
not clear why we need this? maybe we should add a comment here with an example

Reply:
Nice question, I will